test/verifiedTests/base/testIO/testSbmlTestModelToMat.m

```diff
@@ -36,8 +36,14 @@
 assert((exist('Sc_iND750_flux1.xml') == 2));
 cd([CBTDIR, filesep, 'test', filesep, 'verifiedTests', filesep, 'base', filesep, 'testIO']);
 
-% remove the temporary files and 'm_model_collection' folder
-rmdir('m_model_collection', 's');
+% remove the temporary files and 'm_model_collection' folder - loop for windows because apparently 'rmdir' fails sometimes
+for x=1:10
```

Comment:
@jacekwachowiak, what is this?

Reply:
I have found that apparently on windows 'rmdir' fails sometimes - https://www.mathworks.com/matlabcentral/answers/286871-rmdir-frequently-fails-with-the-matlab-rmdir-somedirectoriesnotremoved-error and that a multiple try with a pause helps, otherwise I don't know how to solve it because it should work correctly in theory

Reply:
@jacekwachowiak 
You are currently not catching the error thrown, so your loop doesn't help.
What (could) work is something along the following lines:
```Matlab
isdeleted = false;
while ~isdeleted
  try
    rmdir('m_model_collection','s');
    isdeleted = true;
  catch
    sleep(1); %wait a second before retry
    rehash; %This is mentioned in the answers and might help to bring the matlab path info up to date
  end
end
```

Reply:
Ok, I'll try this